pkg/components/pluggable/discovery.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pluggable
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/dapr/dapr/utils"
+	"github.com/dapr/kit/logger"
```

Comment:
I think this needs to be gofumpt'd with the dapr imports at the end

Reply:
Make lint is not complaining (I guess) and the workflow is passing